quickstart/README.md

```diff
@@ -30,3 +32,35 @@ This invokes the **run.bat** script under *scripts* directory.
 ---
 ### Note
 > If `docker-compose` is not working for JsonBox, please refer to the [instruction](https://github.com/vasanthv/jsonbox#how-to-run-locally) in jsonbox to have it up & running.
+
+## Download Latest Release JAR
+
+### Pre-requisite
+
+- [Restel Jar](https://github.com/techconative/Restel/releases/latest)
```

Comment:
JRE 11 or above is also required..

Reply:
Added JDK for cloning instruction and JRE for JAR instruction.